packages/astro/test/errors.test.js

```diff
@@ -28,10 +28,6 @@ describe('Error display', () => {
 
       // 500 returned
       expect(res.status).to.equal(500);
-
-      // error message includes "unrecoverable error"
-      const body = await res.text();
-      expect(body).to.include('unrecoverable error');
```

Comment:
I think it’s better to leave this test as-is, because it’s technically not testing the original thing anymore (we weren’t able to reliably produce a syntax error. Turning this test into a status check makes it redundant with the runtime test just before it.

Reply:
Or simply remove the syntax error test altogether for now